src/pages/subjects/components/show/SubjectTable.tsx

```diff
@@ -174,13 +289,68 @@ const SubjectTable = () => {
                                 </MenuItem>
                             ))
                         )}
+                        {
+                            <Tooltip title="Add Branch">
+                                <MenuItem
+                                    onClick={() => {
+                                        setAddSemester({
+                                            ...addSemester,
+                                            open: true,
+                                            semEnable: false,
+                                            record: semesterData,
+                                        });
+                                    }}
+                                >
+                                    <AddIcon
+                                        style={{
+                                            color: '#2196f3',
```

Comment:
is there no way to avoid giving this custom styling.
Also please use mui theme variables. dont hardcode the colors. use primary secondary